index.js

```diff
@@ -8,6 +8,7 @@ let glob = require('fast-glob')
 
 let MIXINS_GLOB = '*.{js,json,css,sss,pcss}'
 let IS_WIN = platform().includes('win32')
+let SINGLE_ARGUMENT_KEYWORD = 'asSingleArg'
```

Comment:
Just inline string in the code. Variable is longer and we will not change it a lot.